docs/UserGuide.md

```diff
@@ -345,14 +345,14 @@ a tag with a dash.
 
 :star: **Feature:**
 
-You can add profile picture to your client by storing their photo in `data/images` folder. You should name the photo 
+You can add a profile picture to your client by storing their photo in a new folder named images within the data folder. You should name the photo 
```

Comment:
not sure about this, but should we wrap the folder name (images and data) in a `code` format or some quote?

Reply:
it was in a code format, the guy suggested this instead and I think this will be more readable?

Reply:
I think what Dhafin meant was:
"..in a new folder named `images` within the `data` folder.."